app-doc/gimp-help/gimp-help-2.10.0-r2.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+inherit python-any-r1
+
+DESCRIPTION="GNU Image Manipulation Program help files"
+HOMEPAGE="https://docs.gimp.org/"
+SRC_URI="mirror://gimp/help/${P}.tar.bz2"
+
+LICENSE="FDL-1.2"
+SLOT="2"
+KEYWORDS="~alpha ~amd64 ~hppa ~ia64 ~ppc ~ppc64 ~riscv ~sparc ~x86"
+IUSE=""
+
+BDEPEND="${PYTHON_DEPS}
+	sys-devel/gettext
+"
+
+DEPEND="$(python_gen_any_dep 'dev-libs/libxml2[python,${PYTHON_USEDEP}]')
```

Comment:
I think these are likely both BDEPEND too, and we can update python_check_deps to be -b too then.

Reply:
I hope that I understood correctly this suggestion.
Also add commit with try to fix build env locale.